config/environments/production.rb

```diff
@@ -1,8 +1,10 @@
-require "active_support/core_ext/integer/time"
+require 'active_support/core_ext/integer/time'
 
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  config.action_mailer.default_url_options = { host: 'rails-photoapp.herokuapp.com', port: 3000 }
```

Comment:
Should have the post & port in heroku environment variables so that deploying to another heroku instance will not need a new commit just to update this